openpype/hosts/nuke/plugins/publish/extract_review_baking_streams.py

```diff
@@ -8,15 +8,15 @@
 from openpype.hosts.nuke.api.lib import maintained_selection
 
 
-class ExtractReviewDataMov(publish.Extractor):
+class ExtractReviewDataBakingStreams(publish.Extractor):
     """Extracts movie and thumbnail with baked in luts
 
     must be run after extract_render_local.py
 
     """
 
     order = pyblish.api.ExtractorOrder + 0.01
-    label = "Extract Review Data Mov"
+    label = "Extract Review Data Baking Streams"
```

Comment:
Just wanted to say I got confused by 'streams' as well.

Reply:
Maybe we can have a name like `Extract Review Data Sequences`?

Reply:
Not sure what the 'data' refers to in this case. Also, it works for both mov and sequences, right? Or does it only generates moves? (The `plugin.ExporterReviewMov` plugin it runs in the code does seem to hint it generates movs?)


- `ExtractReviewConversions`? 
- `ExtractConvertReviews`
- `ExtractReviewIntermediates` - if they are never intended to be finals but just preprocessed sequences for further processing?

Reply:
I agree that the current name is not ideal. Initially, it was called "Bake Streams" because it referred to data created in Upstream and related to the Baking of colorspace, which can involve multiple streams.

However, I really like the name "ExtractReviewIntermediates". The only problem I have with it is that the word "Intermediates" can be complex, especially considering that we will be mentioning it frequently every week since we are supporting this feature. This could potentially cause difficulties for non-native speakers. 😄

Reply:
ChatGPT gives me:

> Certainly, here are some simpler words you can use to describe an intermediate file that is meant for processing and not a final output:
> 
> 1. **Working File**: This term suggests that the file is actively being used in the process.
> 
> 2. **Temporary File**: This implies that the file is only temporary and will be discarded after processing.
> 
> 3. **Process File**: This term emphasizes that the file is part of the processing steps.
> 
> 4. **In-Progress File**: This indicates that the file is still being worked on and isn't a final result.
> 
> 5. **Transitory File**: This term implies that the file is transitional and not permanent.
> 
> 6. **Draft File**: This suggests that the file is a preliminary version.
> 
> 7. **Processing File**: This term makes it clear that the file is involved in processing tasks.
> 
> 8. **Intermediate Data**: While it includes "data," it's a clear term for files used during processing.
> 
> 9. **Placeholder File**: This indicates that the file holds a place in the process but isn't final.
> 
> 10. **Provisional File**: This term implies that the file is provisional or temporary in nature.
> 
> Choose the term that best fits your context and your audience's understanding of English.

Still feel like intermediate is the best wording if the files coming out of that extractor aren't intended to be the final output files for exports.

Reply:
@moonyuet I noticed with https://github.com/ynput/OpenPype/pull/5623/commits/6c1e066b3b7d58eba28d38900a886a0c89c556b5 you just changed the name to `ExtractReviewIntermediate`. If the extractor is able to output multiple intermediate files I'd still think the plural for Intermediates would make that clearer? That'd be `ExtractReviewIntermediate` -> `ExtractReviewIntermediates`